llama_index/callbacks/global_handlers.py

```diff
@@ -30,6 +31,8 @@ def create_global_handler(eval_mode: str, **eval_params: Any) -> BaseCallbackHan
         handler = honeyhive_callback_handler(**eval_params)
     elif eval_mode == "promptlayer":
         handler = PromptLayerHandler(**eval_params)
+    elif eval_mode == "deepeval":
+        handler = deepeval_callback_handler(**eval_params)
```

Comment:
could you make a class for deepeval_callback_handler similar to other Handler?

Reply:
I think it's fine to have a function, we do similar for arize and honeyhive (since the integration code lives in their respective libraries)